libbeat/docs/upgrading.asciidoc

```diff
@@ -1,160 +1,324 @@
+--
 
-===== Reference configuration files
+[[migrate-filebeat-registry]]
+==== Migrate {filebeat} registry to use new format
 
-The `<beatname>.full.yml` file, which contains all the non-deprecated
-configuration options is renamed to `<beatname>.reference.yml` starting with
-Beats 6.0. We recommend using this file as a reference only. It's not intended
-to be used in production.
+Starting with version 7.0, {filebeat} stores the registry in a directory
+structure. If Filebeat finds an old registry file at the path set by
+`filebeat.registry.path`, it will automatically migrate the registry file to the
+new format. If the registry file is at a different location, set
+`filebeat.registry.migrate_file` to point to the file.
 
-==== Dashboard upgrades
+[role="xpack"]
+[[ilm-on]]
+==== Check privileges for index lifecycle management (on by default in 7.0)
 
-We recommend re-importing the Kibana dashboards after the Beats and Kibana
-upgrades are complete. This way, you will take advantage of the new Beats
-dashboards created for the 6.0 release.
+// REVIEWERS: not sure if this is worth pointing out in the upgrade docs but
+// I think the default changing is going to confuse some users.
 
-See <<breaking-changes-import-dashboards>> in the Breaking Changes
-documentation.
+Staring with {beats} 7.0, index lifecycle management is on by default when
+sending data to {beats} clusters that support it. Make sure {beats} users have
+the privileges needed to use index lifecycle management, or disable index
+lifecycle management. The following privileges are required:
 
-NOTE: If you've modified the Beats dashboards manually, reimporting them will
-overwrite your changes. Consider saving your dashboards under new IDs, so
-that the reimport doesn't overwrite them.
+* To set up index lifecycle policies: `manage_ilm` and `monitor` on cluster
+* To send data to a secured cluster when index lifecycle management is enabled:
+`manage_index_templates` and `monitor` on cluster, and `index` and `manage` on
+{beats} indices.
 
-==== Heartbeat package names
+// REVIEWERS: The privileges shown here are based on testing I did for
+// https://github.com/elastic/beats/pull/11329, but I am still waiting for
+// more review comments on that PR. 
 
-The DEB and RPM packages for Heartbeat are now named `heartbeat-elastic` to
-avoid conflicts with a different `heartbeat` project.  The `.deb` and `.rpm`
-file names haven't changed, just the package name in the repositories.
+To disable index lifecycle management, set `setup.ilm.enabled: false` in the
+{beats} configuration file.
 
 [[upgrade-mapping-template]]
-=== Upgrade the Elasticsearch mapping template
+=== Upgrade the {es} mapping template
 
-Starting with Beats 6.0, the mapping templates and the default index names are
-versioned. For example, Metricbeat 6.0.0 creates indices like this:
+// REVIEWERS: Does this topic really belong on its own rather than as a
+// sub-topic? Do we want to tell users to set 
```

Comment:
Like topic 'Upgrade Indices' and group ILM, mapping and field changes under this topic?